src/test/java/it/gov/pagopa/rtd/ms/rtdmsdecrypter/service/DecrypterTest.java

```diff
@@ -91,6 +83,32 @@ void shouldDecryptFile() throws IOException, NoSuchProviderException, PGPExcepti
         new BufferedReader(new FileReader(Path.of(resources, "/cleartext.csv").toFile())),
         new BufferedReader(new FileReader(Path.of(resources, "/file.pgp.csv.decrypted").toFile()))));
   }
+
+    @Test
+    void shouldThrowIOExceptionFromMalformedPGPFile() throws IOException, NoSuchProviderException, PGPException {
+
+        // Creates a malformed pgp file (invalid armor)
+        String pgpfile = "-----BEGIN PGP MESSAGE-----\n" +
```

Comment:
@TommasoLencioni Wouldn't it be better to read this value from a static file?

Reply:
Fixed